### DISCUSSION TITLE
Question on font metrics meaning.

### DISCUSSION THREAD

**Discussion by Constantin-Ushkov** [SilenceLeo/snowb-bmf#44] [Q&A] [answered]
Hello, first of all - great work!
Could you pls explain the meaning of some values in exported file?
Well, char id, x, y, width and height - those are self explanatory, but what about other parameters, what is their meaning?
xoffset, yoffset, xadvance, page, chnl... ?
Are there any documentation that I can read about those?
I'm pretty new to this (fonts) field and maybe asking about obvious things. :)

**Comment by SilenceLeo** [comment, marked as answer]
http://www.angelcode.com/products/bmfont/doc/file_format.html

**Reply by Constantin-Ushkov** [reply]
Thank you a lot!
